server/channels/store/sqlstore/channel_bookmark_store.go

```diff
@@ -262,7 +262,7 @@ func (s *SqlChannelBookmarkStore) UpdateSortOrder(bookmarkId, channelId string,
 	}
 
 	bookmarks = utils.RemoveElementFromSliceAtIndex(bookmarks, currentIndex)
```

Comment:
This can be written as `slices.Delete(bookmarks, currentIndex, currentIndex+1)`, which I find hard to read. Hence, I left `RemoveElementFromSliceAtIndex` untouched.

Reply:
Just curious, nothing blocking: do we gain something from using `slices.Delete` in the implementation of `RemoveElementFromSliceAtIndex`?

Reply:
Well, `slices.Delete` does zero the deleted elements from the old slices (https://cs.opensource.google/go/go/+/refs/tags/go1.22.2:src/slices/slices.go;l=227), which slightly speeds up GC.|

I ended up using it instead of the existing custom implementation via [fa4157f](https://github.com/mattermost/mattermost/pull/26620/commits/fa4157faa9604328cf991932b289c43df5c88a58).